mlflow/pyfunc/utils.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import mlflow.pyfunc
```

Comment:
Hmm, I don't think we should have a dependency on pyfunc here. Can we avoid it?

Reply:
Good point, I've removed the `utils` module.